src/libs/focusComposerWithDelay.ts

```diff
@@ -14,21 +14,19 @@ function focusComposerWithDelay(textInput: TextInput | null): FocusComposerWithD
     return (shouldDelay = false) => {
         // There could be other animations running while we trigger manual focus.
         // This prevents focus from making those animations janky.
-        InteractionManager.runAfterInteractions(() => {
```

Comment:
This also caused https://github.com/Expensify/App/issues/27612 where after the emoji picker modal is hidden, focus is put on the composer input immediately even if we closed that modal to open another one, this resulted in a flicker because focus is lost again to the new modal.